Correctly disable screen locking and disable DPMS on XFCE4

jsf9k opened the pull request:
## 🗣 Description ##

This pull request modifies the Ansible code to correctly disable screen locking and DPMS on XFCE4.

## 💭 Motivation and context ##

1. The previous code did not disable `light-locker` in the preferred way.
2. Kali, at least, no longer uses `light-locker` but instead now uses `xfce4-power-manager` and `xfce4-screensaver` to lock the screen and handle DPMS.

## 🧪 Testing ##

All automated tests pass.  I also built a new Kali AMI for our `dev-a` COOL environment as part of cisagov/kali-packer#184 and verified that it no longer locks the screen automatically.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

jsf9k committed:
Fix code that disables screen locking

Kali used to lock the screen using light-locker but now has switched
to using xfce4-power-manager and xfce4-screensaver.

jsf9k committed:
Add a desktop entry file to disable light-locker

This is just in case light-locker is already present in
/etc/xdg/autostart on a platform we support.

mcdonnnj commented on molecule/default/tests/test_default.py:
```diff
@@ -25,3 +26,17 @@ def test_packages(host):
+)
+def test_autostart_files_exist(host, f):
+    """Test that the file exists."""
+    assert host.file(f).exists
```
Just for my own reference do we care about testing file permissions? We've been kind of on and off on this so I'm just looking to reach a consistent opinion.

dav3r replied:
I have no strong opinions either way here.

jsf9k replied:
I added tests for the ownership and mode in commit ca2882a.

mcdonnnj reviewed (approved):
LGTM ✔️ I had one minor question just for consensus.

dav3r reviewed (approved):
Strong work!

jsf9k committed:
Add tests for desktop files' ownership and mode

Co-authored-by: Nick M <50747025+mcdonnnj@users.noreply.github.com>

jsf9k committed:
Add tests for autostart directory

jsf9k committed:
Add desktop-file-utils to package test

jsf9k committed:
Add systemd service test

jsf9k committed:
Use long form permissions to specify modes

Also alphabetize YAML keys.

jsf9k committed:
Add tests for VNC user and user configuration

jsf9k committed:
Exclude service.is_valid test for Ubuntu 22.04

Something funky happens on Ubuntu 22.04.  is_valid is false for
reasons unrelated to the systemd service we put in place.  See #57 for
more details.
